src/spdx3/bump_from_spdx2/annotation.py

```diff
@@ -8,20 +8,24 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from copy import deepcopy
 
 from spdx.model.annotation import Annotation as Spdx2_Annotation
 from spdx3.bump_from_spdx2.message import print_missing_conversion
 from spdx3.model.annotation import Annotation, AnnotationType
 from spdx3.model.creation_information import CreationInformation
+from spdx3.payload import Payload
 
 
-def bump_annotation(spdx2_annotation: Spdx2_Annotation, creation_info: CreationInformation, counter: int) -> Annotation:
+def bump_annotation(spdx2_annotation: Spdx2_Annotation, payload: Payload, creation_info: CreationInformation,
```

Comment:
nit: we incosistently use `creation_info` and `creation_information`. I remember that when I started to implement this I wanted to be consistent with the names for the SPDX3.0 repo and named the class `CreationInformation`, IIRC this is also discussed now, so maybe we can wait with that change and make it consistent once it is decided.

Reply:
Yeah, I stumbled over this, too. Let's wait for the outcome of the renaming discussion.